build/containers/Dockerfile.anomaly-detection-app

```diff
@@ -4,15 +4,15 @@ FROM ${PLATFORM}/debian:bullseye-slim
 # Avoid tzdata prompt 
 ARG DEBIAN_FRONTEND=noninteractive
 
+WORKDIR /app
+COPY ./samples/apps/anomaly-detection-app .
+
 RUN echo "Creating container based on ${PLATFORM}/debian:bullseye-slim" && \
     apt-get update && \
     apt-get install -y protobuf-compiler libprotoc-dev python3-pip \
     python3-grpcio python3-sklearn && \
```

Comment:
Same for `python3-grpcio` and `python3-sklearn`

Reply:
remove